binding.gyp

```diff
@@ -28,7 +28,7 @@
         }
       },
       'xcode_settings': {
-        'CLANG_CXX_LANGUAGE_STANDARD': 'c++11',
+        'CLANG_CXX_LANGUAGE_STANDARD': 'c++14',
```

Comment:
Should/do we need to condition this so it only applies to Node 16?

Reply:
Good question. They obviously build, but we should check if they work on old platforms.

Reply:
I think the other patch that @xzyfer pointed to in the other thread just dropped the line entirely. I wonder if that makes sense either.
I'm good with going with the 14 if it makes sense, since I think you've got more c++ experience than me 😄

Reply:
I'd rather not change the things for previous versions because we know the resulting binaries are problem free and I have the rocker containers to build them ready

Reply:
Our team is using node-sass v4.12.0 and have been unable to compile node-sass.

 We manually changed line 31 to `'CLANG_CXX_LANGUAGE_STANDARD': 'c++14'` and we were able to overcome the 'remove_cv_t' error with xcode.

Reply:
> We manually changed line 31 to `'CLANG_CXX_LANGUAGE_STANDARD': 'c++14'` and we were able to overcome the 'remove_cv_t' error with xcode.

Which node version were you on?

Reply:
Looks like there is canonical answer for which cxx which should be targeting for each Node version at https://github.com/nodejs/node/blob/master/BUILDING.md#official-binary-platforms-and-toolchains

Lots of the decisions in our gyp configuration were made back in the node 0.11  / 0.12 days and cludged on since then to just make it work from them on.

Reply:
> > We manually changed line 31 to `'CLANG_CXX_LANGUAGE_STANDARD': 'c++14'` and we were able to overcome the 'remove_cv_t' error with xcode.
> 
> Which node version were you on?

We are on Node v14.15.1. 

After reading the compatible version table on the readme, we deleted node_modules folder and package-lock.json, upgraded to node-sass 4.14 and still encountered the same error and fixed it with the same solution.